docs/variants/novacustom_ns5x_7x_12th_gen/post_install.md

```diff
@@ -0,0 +1,69 @@
+# Post-installation setup
+
+This document contains extra steps to perform after installing Dasharo in order
+to enable full functionality.
+
+## Touchpad hotkey enablement (Linux)
+
+The touchpad hotkey needs extra setup to function correctly under Linux. To
+enable the touchpad hotkey to work under Linux, follow the steps below:
+
+1. Create a file `/etc/udev/hwdb.d/60-keyboard.hwdb` with the following contents:
+
+    ```bash
+    evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNS50_70MU:*
```

Comment:
Locally I had it fixed... But file didn't save...

Reply:
Can we instead use https://github.com/Dasharo/dasharo-tools/blob/main/clevo/touchpad-fixup ?
Most likely even no adjustments in the script are necesary?

It can be executed with:

```
curl -sSf https://raw.githubusercontent.com/Dasharo/dasharo-tools/main/clevo/touchpad-fixup | sudo sh
```

I wanted to change it for all models.

Reply:
Ideally they would also go here: https://github.com/systemd/systemd/blob/dcf1bf3b6dd934d3af629a1f9f24e083fce85255/hwdb.d/60-keyboard.hwdb#L271

Reply:
@mkopec Please create ticket in dasharo-issue on that.

Reply:
@macpijan  https://github.com/Dasharo/dasharo-issues/issues/218

Reply:
touchpad-fixup will take care of it